website/docs/d/sizes.html.md

```diff
@@ -94,6 +94,13 @@ The following arguments are supported:
   `price_hourly`, or `available`.
 * `values` - (Required) Only retrieves images which keys has value that matches
```

Comment:
```suggestion
* `values` - (Required) Only retrieves sizes which keys has value that matches
```

If we're going to update the docs, might as well fix this up as well.

Reply:
Seems like these got applied and then overwritten?

Reply:
Hmm, I probably didn't pull the changes after the changes were made on GitHub, then I rebased and force pushed.